app/assets/javascripts/analytics/static-analytics.js

```diff
@@ -207,6 +209,26 @@
     this.setDimension(21, position);
   };
 
+  StaticAnalytics.prototype.setTotalNumberOfSections = function() {
+    var sidebarSections = $('.govuk-related-items h2').length;
```

Comment:
Yeah, this would allow us to keep those attributes consistent without blocking changes to presentation (CSS classes) or semantics (h2)